lib/Doctrine/DBAL/Driver/PDOPgSql/Driver.php

```diff
@@ -71,19 +52,19 @@ public function connect(array $params, $username = null, $password = null, array
     /**
      * Constructs the Postgres PDO DSN.
      *
-     * @param array $params
+     * @param mixed[] $params
      *
      * @return string The DSN.
      */
     private function _constructPdoDsn(array $params)
     {
         $dsn = 'pgsql:';
 
-        if (isset($params['host']) && $params['host'] != '') {
+        if (isset($params['host']) && $params['host'] !== '') {
```

Comment:
This can be considered a BC break. Could be replaced with `... && strcmp($params['host'], '') === 0`.